app/controllers/posts_controller.rb

```diff
@@ -33,6 +33,13 @@ def create
     end
```

Comment:
- [ ] On top of your posts controller please add `load_and_authorize_resource` which uses **CanCanCan** for Post authorization in `Post Controller`

Reply:
- Also you have an error on migration regarding with confirmation_token, this is outside of this millstone requirements, but please check into this since migrations should not fail 
![image](https://user-images.githubusercontent.com/62705947/224339785-e3b92753-fdcc-400f-b518-16d7bed6ac43.png)

Reply:
Done and tested.